linux/hci/hci.go

```diff
@@ -302,6 +302,9 @@ func (h *HCI) init() error {
 	WriteLEHostSupportRP := cmd.WriteLEHostSupportRP{}
 	h.Send(&cmd.WriteLEHostSupport{LESupportedHost: 1, SimultaneousLEHost: 0}, &WriteLEHostSupportRP)
 
+	WriteDefaultDataLengthRP := cmd.LEWriteSuggestedDefaultDataLengthRP{}
+	h.Send(&cmd.LEWriteSuggestedDefaultDataLength{SuggestedMaxTxOctets: 251, SuggestedMaxTxTime: 2120}, &WriteDefaultDataLengthRP)
```

Comment:
should these be settable?

Reply:
I'm not sure. It's a suggested value so I think the radio will do what it can and use this value if possible. Maybe I'll make that a follow on improvement.